Sources/PointFree/Footer.swift

```diff
@@ -38,7 +38,7 @@ private let legalView = View<Prelude.Unit> { _ in
     ])
 }
 
-private let pointFreeView = View<Prelude.Unit> { _ in
+private let pointFreeView = View<Prelude.Unit> { _ -> Node in
```

Comment:
was this for compilation times?

I'm starting to think we should get rid of the `View.init` overload that returns a node instead of an array. it isn't the worst to always wrap in an array

Reply:
Had the same thought!